wrappers/wine-msvc.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-EXE="$1"
+EXE=$1
```

Comment:
Why are you removing the quotes here - aren't these necessary? (I do see that I had missed them in the invocation at the bottom, but afaik they should be necessary in both places, right?)

Reply:
> Why are you removing the quotes here - aren't these necessary? (I do see that I had missed them in the invocation at the bottom, but afaik they should be necessary in both places, right?)

According to the [Bash manual](https://www.gnu.org/software/bash/manual/html_node/Shell-Expansions.html#Shell-Expansions), expansions, including word splitting, happen after a command line has been split into tokens (aka lexing). In this case, the the command line is recognized as a single assignment command `a(EXE) <- b($1)` in the interpreter before word splitting, so whether to add a pair of double quotes or not will not affect the meaning of this command.